packages/gestalt/src/Icon/__integration__/Icon.test.js

```diff
@@ -22,8 +22,3 @@ test('Icon uses the dangerouslySetSvgPath prop when icon path is not specified',
   ).toJSON();
   expect(tree).toMatchSnapshot();
 });
-
-test('Icon renders blank when neither icon nor dangerouslySetSvgPath are passed to it', () => {
```

Comment:
This test does not pass Flow since we Flow typed the component to either take one path prop or the other. Some options:
1. Delete this test
2. Add an $ExpectError
3. Fix the Flow typing to not error if neither is passed